source/install-by-copying-files/index.html.md.erb

```diff
@@ -0,0 +1,73 @@
+---
+title: Install by copying files
+weight: 10
+---
+
+# Install by copying files
+
+You can install GOV.UK Frontend by copying our CSS and JavaScript files into your project.
+
+<%= warning_text('We recommend <a href="/installing-with-npm/">installing with Node.js package manager (npm)</a> instead.
+') %>
+
+## What you cannot do
+
+If you install this way, you will not be able to:
```

Comment:
```suggestion
If you install using pre-compiled Sass and JavaScript, you will not be able to:
```

?

Reply:
Good suggestion - but actually, looking at this fresh, I think we can just start with "You will not be able to:"